server/src/test/java/io/crate/integrationtests/DDLIntegrationTest.java

```diff
@@ -438,6 +438,15 @@ public void test_alter_table_drop_constraint_removes_the_constraint_and_leaves_o
                          INTERNAL_ERROR,
                          BAD_REQUEST,
                          4000));
+
+        // Remove last constraint
+        execute("alter table t drop constraint check_qty_gt_zero");
+
+        execute(selectCheckConstraintsStmt);
```

Comment:
Added in order not to go all the way with creating table for dropping one constraint - I think this is just complementary stuff maybe just need to update method name to smth like `can_drop_all_constraints`

Reply:
Can we move this into a dedicated test case? like `test_can_drop_single_constraint` with a comment explaining that this resulting in an issue before..

Reply:
moved in  https://github.com/crate/crate/pull/12996/commits/b2c382f8388700ee12a4389a5acaf41a4a08c00e